dataprep/data_connector/implicit_database.py

```diff
@@ -54,6 +54,7 @@ class ImplicitTable:  # pylint: disable=too-many-instance-attributes
     body_ctype: str
     body: Optional[Fields] = None
     cookies: Optional[Fields] = None
+    pag_params: Dict[str, Any]
```

Comment:
Can you flesh out the Dict into a deterministic class? It is so easy to make mistakes with this loosely defined dict.